biooutput.py

```diff
@@ -0,0 +1,101 @@
+import pandas as pd
+import numpy as np
+
+class Biooutput():
+    """Output Class
+
+    """
+
+    def __init__(self, hr_data3):
+        self.data = hr_data3
+
+    def summarize_ecg(self):
+        """Create txt file summarizing ECG analysis
+
+        :param data: pandas dataframe {'Heartrate', 'B\T', 'time'}
+        :return ecgResults.txt
+        """
+
+        # Finds the last time point in the data file
+        maxTime = self.data['time'].iat[-1]
+
+        # Tells user the purpose of this program
+        printStr = ("This program will summarize the ECG data.\n\
```

Comment:
consider using a triple quoted string literal that spans multiple lines. Also remember to use snake case when naming variables like `print_str`
```python
print_str = """ This is a string.
It spans multiple lines. Looks weird sometimes."""
```

Reply:
Fixed, updated this line and made the variable name camel_case.